### DISCUSSION TITLE
nft-marketplace  => Web3ValidatorError

### DISCUSSION THREAD

**Discussion by Benji09bal** [ConsenSys-archive/truffle#6145] [Web3 Unleashed]
Hello, I have followed "How to Build a NFT Marketplace DApp on Ethereum or Optimism" https://trufflesuite.com/guides/nft-marketplace/. However when deploying locally on ganache or sepolia, I always have this error although I am well connected with the correct wallet address importing on metamask: Web3ValidatorError: Web3 validator found 1 error[s]:
value at "/0" is required. This says it's from pages/create-and-list-nft.js (69:30) @mint > 69 | architectContract.methods.mint(url).send({ from: accounts[0] }).on('receipt', function (receipt) {. I've tried everything and can't find a solution. Could you please tell me 'help?

**Comment by mint** [comment]
stop @ me lol

**Reply by FlaviuWeb3** [reply]
Hi @Benji09bal error[s]: value at "/0" is required," suggests that there is an issue with the contract function call in the pages/create-and-list-nft.js file. Specifically, it seems that the accounts[0] value is missing or not set correctly when trying to mint a new NFT.
